javascript/javascript1/week1/dogAge.js

```diff
@@ -0,0 +1,12 @@
+const dogYearOfBirth=2000;
+const dogYearFuture=2045;
+const dogAge = dogYearFuture-dogYearOfBirth;
+const humanYears=70;
+let shouldShowResultInDogYears;
+if (shouldShowResultInDogYears==true) {
```

Comment:
Try to avoid using `==` for comparison, you should almost always be using `===` for strict comparison. 

See [comparison operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators)